### PR TITLE
Feature/registration and login

### PR DESCRIPTION
**Pull request by DeveloperAspire**

### CODE REVIEW

File: components/Input/index.tsx [hunk 1]
```diff
@@ -1,77 +1,92 @@
...
-	disabled: false,
-	handleShowPassword: () => {},
-	half: false,
-	autoFocus: false,
-	type: 'text',
-	value: '',
-	onChange: () => {},
+	return (
+		<Grid item xs={12} sm={props.half ? 6 : 12}>
+			<FormControl sx={{ width: '100%' }} variant="outlined" error={isError}>
```
Comment (drazomo) [inline components/Input/index.tsx:68]: Is it better if you use TextField instead? It contains all the components/properties you will need that is enwrapped with form control.

https://stackoverflow.com/questions/56122219/in-mui-when-do-we-use-input-vs-textfield-for-building-a-form

Can reuse the code and spread all the props... (It looks like we will use a lot of them 😅)

File: pages/auth/index.tsx [hunk 2]
```diff
@@ -1,62 +1,25 @@
...
-		last_name: '',
-		email: '',
-		password: '',
-		confirmPassword: '',
-	})
 
-	const handleShowPassword = () => setShowPassword(!showPassword)
+import SignIn from '@/components/SignIn'
+import SignUp from '@/components/SignUp'
+import { NextPage } from 'next'
```
Comment (MarkRasavong) [inline pages/auth/index.tsx:14]: 👍 Nice clean logic between the switch of sign/signup

Comment (MarkRasavong) [inline pages/auth/index.tsx:14]: 👍 Nice clean logic between the switch of sign/signup

File: components/Input/index.tsx (hunk 1 above)
Comment (DeveloperAspire) [reply components/Input/index.tsx:68]: Yeah, I get you but I did a little bit of customization for input adornment and error messages, and from the docs here: https://mui.com/material-ui/react-text-field/#input-adornments, I wanted to add so more customization and from the docs, I followed through.

Comment (drazomo) [reply components/Input/index.tsx:68]: Ok I'll merge it. Ill give you a heads up on a update on my branch 👍

Comment (DeveloperAspire) [reply components/Input/index.tsx:68]: What do you mean by reuse?

**Review by drazomo — APPROVED** [review]
Everything is good at this part!

**Pull request merged**